imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/CCITTFaxDecoderStream.java

```diff
@@ -106,6 +111,15 @@ public CCITTFaxDecoderStream(final InputStream stream, final int columns, final
                 "CCITT GROUP 3/4 OPTION UNCOMPRESSED is not supported");
     }
 
+    /**
+     * This is used for CCITT streams from PDF files, which use EncodedByteAlign
+     *
+     * @param b EncodedByteAlign
+     */
+    public void setOptionByteAligned(boolean b) {
```

Comment:
Can this value change while decoding? Otherwise, I'd prefer the field to be final, and set in the constructor only.  And a proper parameter name. :-)